tests/wpt/web-platform-tests/html/semantics/forms/textfieldselection/defaultSelection.html

```diff
@@ -3,33 +3,23 @@
 <title></title>
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
-<textarea>g</textarea>
-<input type="text" value="foo">
-</input>
+<textarea>foo</textarea>
+<input type="text" value="foo"></input>
 <script>
-test(function() {
```

Comment:
@jdm I rebased with upstream, but since i reverted a previous commit, which consisted of this code (which does not exist in master any more), I think this is what is creating problem.

Reply:
Right, I would expect rebasing to give you merge conflicts that need to be resolved. If that doesn't happen, you're not rebasing on top of a recent enough revision.